### PR TITLE
Update manifest as per tiki 27.0 release

### PR DESCRIPTION
**Pull request by rodriguezny**
## Problem

- *New Tiki LTS release available (Tiki 27.0)*

## Solution

- *Update manifest.toml main and lts resource source to Tiki-27.0 *

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
The release cycle question no longer seems relevant now that there is only one version. Is it?

**Comment by rodriguezny** [comment]
> The release cycle question no longer seems relevant now that there is only one version. Is it?

Yes, but once tiki 28 released the release cycle will be relevant.

File: manifest.toml [hunk 1]
```diff
@@ -57,13 +57,13 @@ ram.runtime = "50M"
...
-        url = "https://sourceforge.net/projects/tikiwiki/files/Tiki_26.x_Alnilam/26.2/tiki-26.2.tar.gz/download"
-        sha256 = "d85d513313e2377ecc2fe03f471b053727cb7f9a40143192f95191891557dabf"
+        url = "https://sourceforge.net/projects/tikiwiki/files/Tiki_27.x_Miaplacidus/27.0/tiki-27.0.tar.gz/download"
+        sha256 = "bcc79422239e9b17846082bec4341326518e95a5c058307e61933c2382245dbb"
         format = "tar.gz"
 
         [resources.sources.lts]
-        url = "https://sourceforge.net/projects/tikiwiki/files/Tiki_24.x_Wolf_359/24.6/tiki-24.6.tar.gz/download"
-        sha256 = "1dc1a3f0236c79287b14bfe45eb789e8b0e5c89d2a9089bab496c61a2c73a9f0"
+        url = "https://sourceforge.net/projects/tikiwiki/files/Tiki_27.x_Miaplacidus/27.0/tiki-27.0.tar.gz/download"
```
Comment (marclaporte) [inline manifest.toml]: Folks that choose LTS want stability. I would not replace 24.6 LTS with 27.0 as it's too new. I would wait until 28.0 is released to make LTS branch 27x. In other words, now:

LTS cycle: 24.x (no change)
regular cycle: 26.x -> 27.x
  
And when 28.0 is released:
LTS cycle: 24.x -> 27.x
regular cycle: 27.x -> 28.x

Or just keep 24.x supported until EoL (If it's OK in YunoHost to support more than 2 branches in parallel)

FYI: http://tiki.org/Lifecycle

Comment (rodriguezny) [reply manifest.toml]: Sounds good.
Let's do this:
Now:
LTS cycle: 24.x (no change)
regular cycle: 26.x -> 27.x

And when 28.0 is released:
LTS cycle: 24.x -> 27.x
regular cycle: 27.x -> 28.x

I want to update the code and push the update to this PR.

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**